core/src/main/java/org/apache/calcite/sql/ddl/SqlColumnDeclaration.java

```diff
@@ -48,7 +48,7 @@ public class SqlColumnDeclaration extends SqlCall {
   public final ColumnStrategy strategy;
 
   /** Creates a SqlColumnDeclaration; use {@link SqlDdlNodes#column}. */
-  SqlColumnDeclaration(SqlParserPos pos, SqlIdentifier name,
+  protected SqlColumnDeclaration(SqlParserPos pos, SqlIdentifier name,
```

Comment:
Required for https://github.com/Bodo-inc/Bodo/pull/5520/files#diff-5fe8b785e0ac9316d48a1d302adca47684ef24c7d044afd31c4ad9cc6232340b